### PR TITLE
tiup-mirror: Extend and update

### PR DESCRIPTION
**Pull request by shichun-0415**
### What is changed, added or deleted? (Required)
Update tiup mirror documentation

Closes https://github.com/pingcap/docs/issues/7362
Closes https://github.com/pingcap/tiup/issues/1304

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions (in Chinese)](https://github.com/pingcap/docs-cn/blob/master/CONTRIBUTING.md#版本选择指南).

- [x] master (the latest development version)
- [x] v6.6 (TiDB 6.6 versions)
- [x] v6.5 (TiDB 6.5 versions)
- [x] v6.4 (TiDB 6.4 versions)
- [x] v6.1 (TiDB 6.1 versions)
- [x] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from: https://github.com/pingcap/docs/pull/7367
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

File: tiup/tiup-mirror.md [hunk 1]
```diff
@@ -5,23 +5,21 @@ aliases: ['/docs-cn/dev/tiup/tiup-mirror/','/docs-cn/dev/tiup/tiup-mirrors/','/d
...
 tiup mirror --help
 ```
 
 ```
-The 'mirror' command is used to manage a component repository for TiUP, you can use
+The `mirror` command is used to manage a component repository for TiUP, you can use
 it to create a private repository, or to add new component to an existing repository.
 The repository can be used either online or offline.
-It also provides some useful utilities to help managing keys, users and versions
+It also provides some useful utilities to help manage keys, users, and versions
```
Comment (qqqdan) [inline tiup/tiup-mirror.md:22]:
```suggestion
It also provides some useful utilities to help manage keys, users and versions
```

**Review by qqqdan — APPROVED** [review]

File: tiup/tiup-mirror.md [hunk 2]
```diff
@@ -85,8 +90,117 @@ tiup mirror clone <target-dir> [global-version] [flags]
...
+```bash
+export TIUP_MIRRORS=/shared_data/tiup
+tiup list
+```
+
+设置 `TIUP_MIRRORS` 会永久改变镜像配置，例如 `tiup mirror set`。详情请参考 [tiup issue #651](https://github.com/pingcap/tiup/issues/651)。
+
+### 更新私有仓库
+
+如果使用同样的 `target-dir` 目录再次运行 `tiup mirror clone` 命令，机器会创建新的清单，并下载可用的最新版本的组件。
```
Comment (ran-huang) [inline tiup/tiup-mirror.md:122]: 感觉 `清单` 可能有歧义，manifest 比较准确
```suggestion
如果使用同样的 `target-dir` 目录再次运行 `tiup mirror clone` 命令，机器会创建新的 manifest，并下载可用的最新版本的组件。
```

File: tiup/tiup-mirror.md [hunk 3]
```diff
@@ -85,8 +90,117 @@ tiup mirror clone <target-dir> [global-version] [flags]
...
+
+> **注意：**
+>
+> 重新创建清单之前，请确保所有组件和版本（包括之前下载的早期版本）都包含在内。
```
Comment (ran-huang) [inline tiup/tiup-mirror.md:126]:
```suggestion
> 重新创建 manifest 之前，请确保所有组件和版本（包括之前下载的早期版本）都包含在内。
```

File: tiup/tiup-mirror.md [hunk 4]
```diff
@@ -85,8 +90,117 @@ tiup mirror clone <target-dir> [global-version] [flags]
...
+    ```
+
+    ```
+    The component `hello` version  is not installed; downloading from repository.
+    Starting component `hello`: /home/dvaneeden/.tiup/components/hello/v0.0.1/hello
+    hello
+    ```
 
-使用 `tiup mirror clone` 命令克隆的仓库可以在主机之间共享。可以通过 SCP、NFS 共享文件，也可以通过 HTTP 或 HTTPS 协议使用仓库。用 `tiup mirror set <location>` 命令来指定仓库的位置。
+    执行 `tiup mirror merge` 命令，可以将自定义组件的仓库合并到另一个仓库中。本文假设 `/data/my_custom_components` 中的所有组件都使用 `$USER` 签名：
```
Comment (ran-huang) [inline tiup/tiup-mirror.md:200]:
```suggestion
    执行 `tiup mirror merge` 命令，可以将自定义组件的仓库合并到另一个仓库中。这一操作假设 `/data/my_custom_components` 中的所有组件都使用 `$USER` 签名：
```

**Pull request merged**